packages/Autocompletion.package/ECController.class/instance/currentMenuMorph..st

```diff
@@ -0,0 +1,4 @@
+accessing
+currentMenuMorph: aECMenuMorph
```

Comment:
Why is this called `currentMenuMorph:` and not `menuMorph:` ?
Same with the getter

Reply:
Seems to be due to legacy.
Please remove the legacy calls and refactor to `menuMorph:`